### PR TITLE
feat: validate linked VPs in SIOP flow

### PR DESCRIPTION
**Pull request by nanderstabel**
# Description of change

This feature adds a demo validation of a credential issued by a third-party issuer. The check is done during a SIOP auth request. Visual feedback with issuer logo and checkmark.

## Links to any relevant issues
this change is needed for:
- https://github.com/impierce/identity-wallet/issues/342
- https://github.com/impierce/identity-wallet/issues/344

## How the change has been tested
Unit tests added in `identity-wallet/src/state/did/validate_linked_verifiable_presentations.rs`

This functionality can be manually tested against [this UniCore branch](https://github.com/impierce/ssi-agent/pull/92). Please follow the test instructions there under "How the change has been tested" to spin up a UniCore instance and add a Linked Verifiable Presentation to it.

After setting up the UniCore instance, execute the Verification/siopv2_authorization_requests Request to obtain the Authorization Request string. Copy Paste this string in UniMe (in devmode) to see the result.

## Definition of Done checklist

Add an `x` to the boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: unime/src-tauri/tauri.conf.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   "build": {
     "beforeDevCommand": "npm run dev",
     "beforeBuildCommand": "npm run build",
-    "devUrl": "http://localhost:5173",
+    "devUrl": "http://localhost:4173",
```
Comment (daniel-mader) [inline unime/src-tauri/tauri.conf.json:30]: Why change the dev server port?

File: unime/src/i18n/en/index.ts [hunk 2]
```diff
@@ -288,7 +288,7 @@ const en = {
     },
   },
   DOMAIN_LINKAGE: {
-    TITLE: 'Verified',
+    TITLE: 'Verified website',
```
Comment (daniel-mader) [inline unime/src/i18n/en/index.ts:299]: Is using the term "website" added to make it more relateable for the user? We are not really verifying a website (but the domain), so I think that could be a bit misleading. Shouldn't we just stick with "Verified" and explain more in the dropdown?

**Comment by nanderstabel** [comment]
Putting this back into Draft again since I now have the time to make the backend part of this PR more generic.

**Comment by nanderstabel** [comment]
This PR is now updated to support 'generic' linked VPs. Please check the description for testing instructions :)

File: unime/src/lib/components/StatusIndicator.svelte [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+  import type { ValidationStatus } from '@bindings/user_prompt/ValidationStatus';
+  import { createPopover, melt } from '@melt-ui/svelte';
+
+  import { Image } from '$lib/components';
+  import { hash } from '$lib/utils';
+
+  import Check from '~icons/ph/check-bold';
+  import QuestionMark from '~icons/ph/question-mark-bold';
+  import X from '~icons/ph/x-bold';
```
Comment (maiertech) [inline unime/src/lib/components/StatusIndicator.svelte]: Replace icon imports with

`import {CheckBoldIcon, QUestionMarkBoldIcon, XBoldIcon} from '$lib/icons';`

This was part of a refactoring done while the PR was pending.

File: unime/src-tauri/tauri.conf.json (hunk 1 above)
Comment (maiertech) [reply unime/src-tauri/tauri.conf.json:30]: This was due to a port conflict while developing the ecommerce demo. The ecommerce site was already running on port 5173.

File: unime/src/i18n/en/index.ts (hunk 2 above)
Comment (maiertech) [reply unime/src/i18n/en/index.ts:299]: This was a request by @JelleMillenaar.

**Review by maiertech — APPROVED** [review]
I reveiwed frontend only.

**Review by maiertech — CHANGES_REQUESTED** [review]
Check comment about icon imports. Small fix.

**Review by Oran-Dan — DISMISSED** [review]

File: Cargo.toml [hunk 4]
```diff
@@ -25,7 +25,8 @@ tauri-runtime-wry = { version = "=2.0.0-beta.20" }
 tauri-utils = { version = "=2.0.0-beta.19", features = [ "resources" ] }
 tauri-winres = "=0.1"
 
-did_manager = { git = "https://github.com/impierce/did-manager.git", rev = "2b88f55" }
+agent_shared = { git = "https://git@github.com/impierce/ssi-agent.git", rev = "1823810" }
```
Comment (daniel-mader) [inline Cargo.toml]: Whoop whoop! 🎉

**Review by daniel-mader — APPROVED** [review]

**Review by maiertech — APPROVED** [review]

**Pull request merged**